config/servers.yml

```diff
@@ -91,6 +91,7 @@
 #        autoCommitEnabled:         true
 #        queueSize:                 1000
 #        fetchSize:                 1048576
+#        minPartitionCount:         1
```

Comment:
Does this `minPartitionCount` need to be deployment property so that it can be specified for specific stream deployment rather than message bus specific property?
Also, lets say if someone needs to have higher `minPartitionCount` for a given stream but a very low value for other, then having this at the message bus level may not fit well. what do you think?